docs/dsl_resource.md

```diff
@@ -98,3 +98,23 @@ end
 ```
 
 For a full example, see our [example resource](https://github.com/chef/inspec/blob/master/examples/profile/libraries/example_config.rb).
+
+## Lazy Loading
+
+As of InSpec v4.16, resources are lazily loaded into the `inspec`
+process upon use. This greatly speeds up the initial startup costs of
+the `inspec` process and only loads what you need to use (eg, `inspec
+--version` no longer runs for 10 seconds!).
+
+It does change the way the resource registry is handled in ways that
+might break some assumptions. Specifically `inspec.<resource>` isn't
+pre-populated with the core resources that InSpec ships with. If you
+make a local/custom resource of the same name, referring to the core
+resource via `inspec.<resource>` will not resolve to the core
+resource.
+
+If you do need to do this, it is better to make a local resource with
```

Comment:
What is "this" supposed to refer to in this sentence?

Reply:
handled via a new subsection heading.